lib/slayer/form.rb

```diff
@@ -1,17 +1,31 @@
+require 'virtus'
+require 'dry-validation'
+
 module Slayer
   class Form
-    extend ModelAttribute
-    include ActiveModel::Validations if defined?(Rails)
+    attr_reader :errors
+    include Virtus.model
 
-    # Internal: Form Class Methods
     class << self
-      def from_params(params)
-        raise NotImplementedError
-      end
+      attr_accessor :_schema
 
-      def from_model(model)
-        raise NotImplementedError
+      def validations(&block)
+        self._schema = Dry::Validation.Schema(&block)
```

Comment:
Sorry, I believe the above would need to put it in a hash based on the class:

```
  @@schema ||= {}
  @@schema[self] = Dry::Validation.Schema(&block)
```

This makes the accessing code grosser than it currently is, but would have the benefit of leaving less 'cruft' on the subclass.